### DISCUSSION TITLE
Problem with creating a session

### DISCUSSION THREAD

**Discussion by erkaner** [ether/etherpad-lite#5043] [Q&A]
In my web application, I try to integrate etherpad. When users are logged into the system first time, if they do not have an etherpad author account, this is created. Then, their authorID is recorded in the db of my application. the No problems with that. Then, I try to create a session on a separate page by retreiving the authorID from the database. However. I receive "authorID does not exist" error. I wonder if the approach I followed is incorrect? Any ideas why this is not working?

**Comment by webzwo0i** [comment]
Can you write exact steps to replicate your problem?
There was a similar [bug](https://github.com/ether/etherpad-lite/issues/5000) some weeks ago that is fixed in latest develop.

**Reply by erkaner** [reply]
It was my mistake. Sorry!
